assets/src/design-system/components/toggle/index.js

```diff
@@ -143,7 +141,7 @@ const ToggleContainer = styled.div(
         }
 
         ~ ${Circle}:after {
-          transform: translate3d(${CIRCLE_FINAL_POSITION}px, 0, 0);
+          left: ${CIRCLE_FINAL_POSITION}px;
```

Comment:
😭

Reply:
Why was this needed? Is the property otherwise not automatically flipped?

Reply:
Yeah exactly, the property was not automatically flipped so it looked like this:
![rtl toggle.gif](https://images.zenhubusercontent.com/5e9dcd9a9ff5dfdaffcdf132/03de986b-3579-4400-b730-431268211806)

It would be preferable to keep it how it was but the `translate3d` wasn't translating in the opposite direction